core/loader/utils.ts

```diff
@@ -61,6 +61,36 @@ export function findAnchorHrefs(
     .map(a => buildFullURL(a, text.url))
 }
 
+/**
```

Comment:
Very good comment. But we don’t use it in another functions.

Can we keep the old comment-less format in PR?

But I am open to discuss adding more comments (just prefer to have some policy).